server/src/main/java/org/elasticsearch/action/bulk/BulkRequest.java

```diff
@@ -369,7 +369,7 @@ public BulkRequest add(BytesReference data, @Nullable String defaultIndex, @Null
                             } else if (SOURCE.match(currentFieldName, parser.getDeprecationHandler())) {
                                 fetchSourceContext = FetchSourceContext.fromXContent(parser);
                             } else {
-                                throw new IllegalArgumentException("Action/metadata line [" + line + "] contains an unknown parameter [" + currentFieldName + "]");
+                                throw new IllegalArgumentException("GenericAction/metadata line [" + line + "] contains an unknown parameter [" + currentFieldName + "]");
```

Comment:
I think this was right before? Or, at least, this change doesn't make it more right.